package.json

```diff
@@ -118,12 +118,12 @@
     "env:stop": "./bin/local-env/stop.sh",
     "env:reset-site": "./bin/local-env/install-wordpress.sh --reset-site",
     "lint": "npm-run-all --parallel lint:*",
-    "lint:css": "./bin/get-changed-files.sh '\\.s?css$' | xargs -r wp-scripts lint-style",
+    "lint:css": "git diff HEAD^ --diff-filter=d --name-only | grep -E '\\.s?css$' | xargs -r wp-scripts lint-style",
     "lint:css:fix": "npm run lint:css -- --fix",
-    "lint:js": "./bin/get-changed-files.sh '\\.js$' | xargs -r wp-scripts lint-js",
+    "lint:js": "git diff HEAD^ --diff-filter=d --name-only | grep -E '\\.js$' | xargs -r wp-scripts lint-js",
     "lint:js:fix": "npm run lint:js -- --fix",
-    "lint:php": "./bin/get-changed-files.sh '\\.php$' | xargs -r ./vendor/bin/phpcs",
-    "lint:php:fix": "./bin/get-changed-files.sh '\\.php$' | xargs -r ./bin/phpcbf.sh",
+    "lint:php": "git diff HEAD^ --diff-filter=d --name-only | grep -E '\\.php$' | xargs -r ./vendor/bin/phpcs",
```

Comment:
Why `HEAD^` and not `HEAD`?

Also, invoking via `xargs` is going to mean that we won't be able to take advantage of `<arg name="parallel" value="20"/>` in the PHPCS config.

Reply:
> Why `HEAD^` and not `HEAD`?

If I recall correctly it was because of renamed files not being listed when `HEAD` was used.

Nevertheless, it wouldn't be feasible changing these lint scripts here. It would break the workflow of linting the whole project (like in a CI environment, for example).

As discussed a few days ago, I think we can take advantage of a package called [lint-staged](https://github.com/okonet/lint-staged), which gives us fine-grained control on what commands to run when certain files are staged. This would allow us to keep our original lint scripts intact, and with the combination of a pre-commit hook, we could have file changes being automatically linted whenever we're committing, for example.

See https://github.com/ampproject/amp-wp/pull/4441/commits/ab034bce6c1a879c07fedcd4b943ed1413bae14d for a basic setup of what I think is sufficient for the project.